module/documents/actor/actor.mjs

```diff
@@ -277,6 +277,19 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
 
   /* -------------------------------------------- */
 
+  /**
+   * Calculate the bonus from any cover the actor is affected by.
+   * @returns {number}      The cover bonus to AC and dexterity saving throws.
+   */
+  getCoverBonus() {
+    const { coverHalf, coverThreeQuarters } = CONFIG.DND5E.statusEffects;
+    if ( this.statuses.has("coverThreeQuarters") ) return coverThreeQuarters.coverBonus;
+    else if ( this.statuses.has("coverHalf") ) return coverHalf.coverBonus;
```

Comment:
Just in case someone removes these conditions.

```suggestion
    if ( this.statuses.has("coverThreeQuarters") ) return coverThreeQuarters?.coverBonus;
    else if ( this.statuses.has("coverHalf") ) return coverHalf?.coverBonus;
```